osu.Game/Rulesets/Judgements/Judgement.cs

```diff
@@ -190,10 +190,9 @@ public static int ToNumericResult(HitResult result)
                     return 200;
 
                 case HitResult.Great:
-                    return 300;
-
+                // Perfect doesn't actually give more score / accuracy directly.
                 case HitResult.Perfect:
-                    return 315;
+                    return 300;
```

Comment:
I don't really feel good about this and what it means for the existence of numeric judgement results as a whole...